### ISSUE TITLE
Wish: allow model_profile to specify only some variable_splits (python)

### ISSUE THREAD

**Issue by mayer79** [ModelOriented/DALEX#470]
I am using dalex to plot multiple partial dependence plots in Python. It is a very useful package!

For some numeric features (especially those with outliers), I would like to specify the evaluation grid using option `variable_splits` in `model_profile`. However, if I do this, I need to specify a grid for all other features as well. Ideally, the function would use the keys in `variable_splits` only to overwrite the default. For those features without specified `variable_splits` key, it would resort to the default.

In the following case: the left-most feature should stop at a maximum of 10. The range for the other feature is fine though:

![image](https://user-images.githubusercontent.com/16206095/137584291-8168295b-5df6-4b28-a605-06989bd62799.png)

**Comment by hbaniecki** [comment]
Agreed, but it will change the default behaviour so it probably should be added in `v1.5.0`.

**Comment by hbaniecki** [comment]
Actually, I am not sure if it makes sense for the keys of `variable_splits` to override `variables=None` (like it is now). This behaviour was probably copied from R. I will investigate it.
